### PR TITLE
Make the encryption:encrypt-all command to accept "-y" as input option

### LINKED ISSUE #33: missing -y option in  occ encryption:encrypt-all -y
### Expected behavior:

To create a script to encrypt automatically:

``` 
#!/bin bash
occ app:enable encryption
occ encryption:enable
occ encryption:select-encryption-type masterkey -y
occ encryption:encrypt-all -y
```

### Actual behavior:

Manually works except for the last command:

```
# occ app:enable encryption
encryption enabled
# occ encryption:enable
Encryption enabled

Default module: OC_DEFAULT_MODULE
# occ encryption:select-encryption-type masterkey -y
Master key successfully enabled.
occ encryption:encrypt-all -y

                                   
  The "-y" option does not exist.  
                                   

encryption:encrypt-all
```

This -y option is possible with other commands, for example:
```
occ encryption:select-encryption-type masterkey -y
```

it would be good to have the option `-y` :
```
occ encryption:encrypt-all -y
``` 

it makes no sense to have the options ` -n ` and  `-q ` if there is no ` -y ` 
```
# occ encryption:encrypt-all --help
Usage:
  encryption:encrypt-all

Options:
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
      --no-warnings     Skip global warnings, show command output only
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

Help:
  This will encrypt all files for all users. Please make sure that no user access his files during this process!
```

### PR DESCRIPTION
**Pull request by pako81**
## Related Issue
- Fixes https://github.com/owncloud/encryption/issues/33 and https://github.com/owncloud/enterprise/issues/339

## How Has This Been Tested?
Manually by running the encryption:encrypt-all -y command and see that files get correctly encrypted

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:
- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: core/Command/Encryption/EncryptAll.php [hunk 1]
```diff
@@ -99,13 +100,20 @@ protected function configure() {
...
+			'Answer yes to all questions'
+		);
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output) {
 		if ($this->encryptionManager->isEnabled() === false) {
 			throw new \Exception('Server side encryption is not enabled');
 		}
-
+
```
Comment (micbar) [inline core/Command/Encryption/EncryptAll.php]: @pako81 Please remove this additional linebreak, it lets the coding standard test fail.
You could check this before pushing with `make test-php-style` 😄

**Review by micbar — DISMISSED** [review]

**Comment by micbar** [comment]
Let us see what the test coverage says.

File: core/Command/Encryption/EncryptAll.php (hunk 1 above)
Comment (pako81) [reply core/Command/Encryption/EncryptAll.php]: done

**Review by sharidas — APPROVED** [review]
LGTM :+1:

File: core/Command/Encryption/EncryptAll.php [hunk 2]
```diff
@@ -99,13 +100,19 @@ protected function configure() {
...
+			'Answer yes to all questions'
+		);
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output) {
 		if ($this->encryptionManager->isEnabled() === false) {
 			throw new \Exception('Server side encryption is not enabled');
-		}
-
+		}
```
Comment (patrickjahns) [inline core/Command/Encryption/EncryptAll.php]:
```
   1) core/Command/Encryption/EncryptAll.php (class_definition, no_trailing_whitespace, braces)
      ---------- begin diff ----------
--- Original
+++ New
@@ -111,7 +111,7 @@
        protected function execute(InputInterface $input, OutputInterface $output) {
                if ($this->encryptionManager->isEnabled() === false) {
                        throw new \Exception('Server side encryption is not enabled');
-               }       
+               }
                $yes = $input->getOption('yes');
                $masterKeyEnabled = $this->config->getAppValue('encryption', 'useMasterKey', '');
                $userKeyEnabled = $this->config->getAppValue('encryption', 'userSpecificKey', '');

      ----------- end diff -----------


Checked all files in 1.157 seconds, 14.000 MB memory used
make: *** [test-php-style] Error 8

```

**Review by patrickjahns — APPROVED** [review]
LGTM

**Comment by phil-davis** [comment]
Backports:
`stable10` #35605 
`release-10.2.1 #35606

**Pull request merged**